_pytest/python.py

```diff
@@ -1849,12 +1849,10 @@ def toterminal(self, tw):
         for tbline in self.tblines:
             tw.line(tbline.rstrip())
         lines = self.errorstring.split("\n")
-        for line in lines:
-            if line == lines[0]:
-                prefix = 'E       '
-            else:
-                prefix = '        '
-            tw.line(prefix + line.strip(), red=True)
+        if lines:
+            tw.line('E       ' + lines[0].strip(), red=True)
```

Comment:
Done.  Not sure though if that is really good like that.
After all it should be some more generic method probably for this kind of messages?